RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -399,3 +398,92 @@ void auto_useBlackMonolith()
 	// use monolith
 	visit_url("campground.php?action=monolith");
 }
+
+boolean auto_haveAugustScepter()
+{
+	static item scepter = wrap_item($item[august scepter]);
+	if(auto_is_valid(scepter) && (item_amount(scepter) > 0 || have_equipped(scepter)))
+	{
+		return true;
+	}
+	return false;
+}
+
+void auto_scepterSkills()
+{
+	if(!auto_haveAugustScepter())
+	{
+		return;
+	}
+	//Day 1 skills
+	if(my_daycount() == 1)
+	{
+		//Get some stats
+		if(my_primestat() == $stat[Muscle])
+		{
+			if(canUse($skill[Aug. 12th: Elephant Day!]) && !get_property("_aug12Cast").to_boolean())
+			{
+				use_skill($skill[Aug. 12th: Elephant Day!]);
+			}
+		}
+		if(my_primestat() == $stat[Mysticality])
+		{
+			if(canUse($skill[Aug. 11th: Presidential Joke Day!]) && !get_property("_aug11Cast").to_boolean())
+			{
+				use_skill($skill[Aug. 11th: Presidential Joke Day!]);
+			}
+		}
+		if(my_primestat() == $stat[Moxie])
+		{
+			if(canUse($skill[Aug. 23rd: Ride the Wind Day!]) && !get_property("_aug23Cast").to_boolean())
+			{
+				use_skill($skill[Aug. 23rd: Ride the Wind Day!]);
+			}
+		}
+		if(canUse($skill[Aug. 24th: Waffle Day!]) && !get_property("_aug24Cast").to_boolean())
+		{
+			use_skill($skill[Aug. 24th: Waffle Day!]); //get some waffles to hopefully change some bad monsters to better ones
+		}
+		if(canUse($skill[Aug. 30th: Beach Day!]) && !get_property("_aug30Cast").to_boolean())
+		{
+			use_skill($skill[Aug. 30th: Beach Day!]); //Rollover adventures
+		}
+		if(canUse($skill[Aug. 28th: Race Your Mouse Day!]) && !get_property("_aug28Cast").to_boolean() && ((!auto_hasStillSuit() && item_amount($item[Astral pet sweater]) == 0) || in_small()))
+		{
+			if(!is100FamRun())
+			{
+				handleFamiliar("stat"); //get any familiar equipped if not in a 100% run
+			}
+			use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment
+		}
+	}
+	//Day 2+ skills
+	if(my_daycount() >= 2)
+	{
+		if(canUse($skill[Aug. 24th: Waffle Day!]) && !get_property("_aug24Cast").to_boolean())
+		{
+			use_skill($skill[Aug. 24th: Waffle Day!]); //get some waffles to hopefully change some bad monsters to better ones
+		}
+		if(canUse($skill[Aug. 28th: Race Your Mouse Day!]) && !get_property("_aug28Cast").to_boolean() && ((!auto_hasStillSuit() && item_amount($item[Astral pet sweater]) == 0) || in_small()))
+		{
+			if(!is100FamRun())
+			{
+				handleFamiliar("stat"); //get any familiar equipped if not in a 100% run
+			}
+			use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment
+		}
+	}
+}
+
```

Comment:
Would make sense to add the clover one to both `cloversAvailable()` and `cloverUsageInit()`. This would mean we would have 6 uses for D1. 4 for above, 1 for stomach, 1 for clovers.

Perhaps also move the stat ones to the power leveling code. Only use if we are going to power level? Currently will always give 50 substats since will be done at start of day I think. Moving it makes sense as better to use as needed for this one and then free's up a cast for other stuff

Reply:
Makes sense